diesel_cli/tests/migration_redo.rs

```diff
@@ -90,3 +90,47 @@ Running migration 12345_create_users_table
         result.stdout()
     );
 }
+
+#[test]
+fn output_contains_path_to_migration_script() {
+    let p = project("output_contains_path_to_migration_script")
+        .folder("migrations")
+        .build();
+    p.create_migration(
+        "output_contains_path_to_migration_script",
+        "CREATE TABLE users (id INTEGER PRIMARY KEY);",
+        "DROP TABLE users};",
+    );
+
+    // Make sure the project is setup
+    p.command("setup").run();
+
+    let result = p.command("migration").arg("redo").run();
+
+    assert!(!result.is_success(), "Result was successful {:?}", result);
+    assert!(
+        result.stdout().contains("down.sql"),
+        "Unexpected stdout {}",
+        result.stdout()
+    );
+}
+
+#[test]
+fn error_migrations_fails() {
+    let p = project("redo_error_migrations_fails")
+        .folder("migrations")
+        .build();
+    p.create_migration(
+        "redo_error_migrations_fails",
+        "CREATE TABLE users (id INTEGER PRIMARY KEY);",
+        "DROP TABLE users};",
+    );
+
+    // Make sure the project is setup
+    p.command("setup").run();
+
+    let result = p.command("migration").arg("redo").run();
+
+    assert!(!result.is_success());
+    assert!(result.stdout().contains("Failed with: "));
```

Comment:
I'd look for part of the syntax error instead

Reply:
The text of the syntax error is different for different databases, so I did so. How can I do it better and do we need syntax error text in tests?

Reply:
Nah that's fine by me